gns3server/schemas/iou.py

```diff
@@ -182,7 +186,11 @@
         "command_line": {
             "description": "Last command line used by GNS3 to start QEMU",
             "type": "string"
-        }
+        },
+        "application_id": {
+            "description": "Application ID for running IOU image",
+            "type": ["integer", "null"]
```

Comment:
We support old behavior (local generation) still, but of course we can force to define application_id.